packages/dashboard/src/customization/widgets/types.ts

```diff
@@ -71,6 +72,13 @@ export type BarChartProperties = QueryProperties & {
 export type TableProperties = QueryProperties & {
   thresholdSettings?: ThresholdSettings;
   fontSettings?: ComplexFontSettings;
+  experimentalMode?: boolean;
```

Comment:
do we need this property? this will be part of the public api

Reply:
removed this flag